x/liquidstaking/spec/01_concepts.md

```diff
@@ -4,8 +4,38 @@
 
-Delegators who stake their assets through liquid staking module will get bTokens (pegged staked representative tokens) which can used for additional utilities. When delegators want to unstake, the module will burn the bTokens and give them native tokens back.
+## Liquid Validators
 
-Voters holding bToken or farming positions, pool coins including bToken, will have the corresponding voting power when voting for governance.
+Liquid validators are determined through governance process. They must be defined in governance parameter called WhitelistedValidators and as long as they comply the active condition (you can find more details about what active condition is in the next document), they become liquid validators. When users liquid stake or unstake, there is a proxy account that acts as a middleman. When users liquid stake, the amount of liquid staking is transferred to the proxy account which distributes amount to all active liquid validators in correspondence with their weight. When users liquid unstake, the proxy account undelegates weighted shares from the active liquid validators. Note that gas usage can increase as there are more liquid validators. 
+
+## Liquid Governance
+
+The module acknowledges voting power of all voters by aggregating the core activities that is related to the bToken. These are the following core activities:
+
+- Balance of `bToken`
+- Balance of `PoolCoin(s)` that includes `bToken`
+- Farming position of `bToken`
+- Farming position of `PoolCoin(s)` that include `bToken`
+
+## Rebalancing
+
+The module rebalances liquid tokens of active liquid validators by redelegating from one liquid validator to another. Some cases include when there is a change in whitelisted validators and a liquid validator gets slashed. Technically, it is worth noting that some redelegation may fail due to redelegation hopping restriction in the staking module of Cosmos SDK. In that case, the module retries at the beginning of next block until it gets resolved.
+
+A redelegation object is created every time a redelegation occurs. To prevent "redelegation hopping" redelegations may not occur under the situation that:
+
+- the (re)delegator already has another immature redelegation in progress with a destination to a validator (let's call it Validator X)
+- and, the (re)delegator is attempting to create a new redelegation where the source validator for this new redelegation is Validator X.
+
+## Restake
+
+The module restakes amount to all active liquid validators that corresponds to their weight when an accumulated reward is over `RewardTrigger` value. 
+
+## Unbonding Period
+
+Liquid stakers who unbond their delegation must wait for the duration of the `UnStakingTime`. It is a chain-specific parameter. During the unbonding period, they are still exposed to being slashed for any liquid validator’s misbehavior.
```

Comment:
Not sure, but:

```suggestion
Liquid stakers who unbond their delegation must wait for the duration of the `UnbondingTime`. It is a chain-specific parameter. During the unbonding period, they are still exposed to being slashed for any liquid validator’s misbehavior.
```

Reply:
I don't even know where `UnStakingTime` comes from. Thanks! Let me change that.